src/ProcessManager/ProcessManager.php

```diff
@@ -0,0 +1,321 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Synolia\SyliusAkeneoPlugin\ProcessManager;
+
+use Symfony\Component\Process\Exception\ProcessTimedOutException;
+use Symfony\Component\Process\Process;
+
+/**
+ * The process manager for executing multiple processes in parallel.
```

Comment:
Can you indicate that we take it from BluePsyduckProcessManager ?

Reply:
i was sure a `@package BluePsyDuck` appeared at the top of the file but it's not :face_with_diagonal_mouth: